pkgs/misc/emulators/citra/base.nix

```diff
@@ -0,0 +1,215 @@
+{ pname, version, src, branchDesc, branchName
+, lib, stdenv, config, fetchFromGitHub
+, catch2, cmake, ninja, pkg-config, wrapQtAppsHook
+, enet, fmt, inih, libressl, soundtouch, zstd, discord-rpc
+, boost173, ffmpeg, libusb1, qtbase, qtdeclarative, qtmultimedia, qttools, rapidjson, SDL2
+, alsaSupport      ? stdenv.isLinux,                      alsa-lib      ? null
+, discordSupport   ? true
+, fdkSupport       ? true,                                fdk_aac       ? null
+, jackaudioSupport ? true,                                libjack2      ? null
+, onlineSupport    ? true
+, pulseSupport     ? config.pulseaudio or stdenv.isLinux, libpulseaudio ? null
+, sndioSupport     ? true,                                sndio         ? null
+, udevSupport      ? stdenv.isLinux,                      udev          ? null
```

Comment:
Do we need the `? null` here? I think it should work fine without.

Reply:
Corrected, thanks.